srcpkgs/ddcutil/template

```diff
@@ -28,3 +28,26 @@ esac
 pre_configure() {
 	NOCONFIGURE=1 ./autogen.sh
 }
+
+libddcutil_package() {
+	conflicts="ddcutil<=2.1.4_1"
+	short_desc+=" - library files"
+	pkg_install() {
+		vmove "usr/lib/*.so.*"
+		vmove "usr/share/ddcutil"
+		vmove "usr/lib/modules-load.d"
+		vmove "usr/lib/udev"
```

Comment:
```suggestion
```
shouldn't these stay in the main pkg, as `ddcutil` needs them and `libddcutil` is not a dependancy of `ddcutil`, thus breaking existing setups